haiku-apps/wolle/wolle-0.1~git.recipe

```diff
@@ -0,0 +1,45 @@
+srcGitRev="be9165f52bee14d3cd1d19600ce6cfa44f217df4"
+SUMMARY="A screensaver displaying  woollen string"
+DESCRIPTION="Wolle is a screensaver which draws a woollen string over 
+your desktop"
+HOMEPAGE="https://github.com/HaikuArchives/Wolle"
+COPYRIGHT="2001 - 2008 , Werner Freytag.
+2009, Haiku.
+"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/localanu/Wolle/archive/$srcGitRev.tar.gz" #temporarily,waiting my pr merged
+CHECKSUM_SHA256="1eed342ebfcd0615b57b50150468726e2c62898c47d8ff1ec91d3ba23884d6c5"
+SOURCE_FLIENAME="Wolle-$srcGitRev.tar.gz"
```

Comment:
```diff
+srcGitRev="21c02f8a85d290ae3a16da5d494a31ae19df37f6"
-SOURCE_URI="https://github.com/localanu/Wolle/archive/$srcGitRev.tar.gz" #temporarily,waiting my pr merged
+SOURCE_URI="https://github.com/HaikuArchives/Wolle/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="1eed342ebfcd0615b57b50150468726e2c62898c47d8ff1ec91d3ba23884d6c5"
+CHECKSUM_SHA256="291a7a62679c546653f2db120addd4859de05c14401de044845bb06553a666ad"
-SOURCE_FLIENAME="Wolle-$srcGitRev.tar.gz"
+SOURCE_FILENAME="Wolle-$srcGitRev.tar.gz"
 SOURCE_DIR="Wolle-$srcGitRev"
```
(F**LI**ENAME -> F**IL**ENAME)

Reply:
@localanu, when you move the `wolle` directory from `haiku-apps` to `haiku-misc`, as @scottmc requested, could you please **_also_** rename the recipe to `wolle-2.0~git.recipe` ? Thanks!
For your convenience, here are the commands you can copy-paste:
```
cd /boot/home/haikuports
mkdir haiku-misc/wolle
git mv haiku-apps/wolle/wolle-0.1~git.recipe haiku-misc/wolle/wolle-2.0~git.recipe
git commit haiku-apps/wolle/wolle-0.1~git.recipe haiku-misc/wolle/wolle-2.0~git.recipe
rmdir haiku-apps/wolle
```